_bibliography/references.bib

```diff
@@ -731,6 +763,19 @@ @article{JAHJKQV19
   website   = {https://moves-rwth.github.io/prophesy/}
 }
 
+@inproceedings{SJK19,
+  author       = {Jip Spel and
+                  Sebastian Junges and
+                  Joost{-}Pieter Katoen},
+  title        = {Are Parametric Markov Chains Monotonic?},
+  booktitle    = {{ATVA}},
+  series       = {Lecture Notes in Computer Science},
```

Comment:
Very nitpicky, but "LNCS"?

Reply:
Yes, although there were 13 occurrences of Lecture Notes... :-)

Reply:
I have a script I can run over it ;)